client/js/lounge.js

```diff
@@ -275,7 +286,7 @@ $(function() {
 
 			chat.find(".chan.active .chat").trigger("msg.sticky"); // fix growing
 		})
-		.tab(completeNicks, {hint: false})
+		.tab(completeNicks(false), {hint: false})
```

Comment:
Whateveeeeer. Let's just make this happen, please.
@xPaw, feel free to commit your change on this PR.